eql/parser.py

```diff
@@ -1023,6 +1023,20 @@ def subquery_by(self, node, num_values=None, position=None, close=None, allow_fo
             raise self._error(error_node, message, num=num_values, end=end)
 
         kwargs = {}
+
+        repeated_sequence = node["repeated_sequence"]
+
+        runs_count = 1
+        if repeated_sequence is not None:
+            runs_count = node["repeated_sequence"]["UNSIGNED_INTEGER"].value
```

Comment:
better to cast it to an int here to keep type consistent? As it is, it could return an int or str

```suggestion
            runs_count = int(node["repeated_sequence"]["UNSIGNED_INTEGER"].value)
```